guides/common/modules/proc_creating-an-operating-system-for-debian.adoc

```diff
@@ -14,15 +13,23 @@ endif::[]
 
 .Procedure
 . In the {ProjectWebUI}, navigate to *Hosts > Operating Systems*.
-. Click *Create Operating System* to create an operating system entry for Debian.
-. Enter `Debian 11` as *Name* for the operating system.
+. Click *Create Operating System* to create an operating system entry for {os_name}.
+. Enter `{os_name} {os_major}` as *Name* for the operating system.
 Choose a name as reported by Ansible, Puppet, or Salt as fact.
-. Set the *Major Version* to `11` and omit the *Minor Version*.
-. Set the *Release Name* to `bullseye` for Debian 11 or `focal` for Ubuntu 20.04.
+. Set the *Major Version* to `{os_major}` and omit the *Minor Version*.
+ifeval::["{os_name}" == "Ubuntu"]
++
+[CAUTION]
+====
+If you set a minor version, you cannot provision hosts running Ubuntu 22.04 using Autoinstall.
```

Comment:
You actually can (and should) set a minor version, but it must be set correctly.
For example with `Ubuntu 20.04`, some users would set `20` as a major and `04` as a minor. But it must be `20.04` as a major and then the corresponding sub-releases as a minor (e.g. `1`, `2`, `3`..). Starting with `Ubuntu 20.04.3`, the new Autoinstall mechanism must be used and we evaluate the major/minor field in order to identify this.
So, I would suggest something like the following:

```suggestion
If you create an operating system entry for Ubuntu in {Project}, it is important to pay attention to the versioning. For example in Ubuntu 20.04, the major field must be set to 20.04. Only in case of a follow-up release like 20.04.3, the major version is kept as 20.04 and the minor version is set to 3.
```